src/test/java/net/sf/jabref/gui/journals/ManageJournalAbbreviationsTest.java

```diff
@@ -0,0 +1,560 @@
+
+        viewModel.saveJournalAbbreviationFiles();
+
+        String expected = "Abbreviations = Abb" + NEWLINE + "Test Entry = TE" + NEWLINE + "MoreEntries = ME" + NEWLINE
+                + "JabRefTestEntry = JTE" + NEWLINE + "";
+        String actual = Files.contentOf(new File(testFile4Entries), StandardCharsets.UTF_8);
+        Assert.assertEquals(expected, actual);
+
+        expected = "Abbreviations = Abb" + NEWLINE + "Test Entry = TE" + NEWLINE + "MoreEntries = ME" + NEWLINE
+                + "SomeOtherEntry = SOE" + NEWLINE + "";
+        actual = Files.contentOf(new File(testFile5EntriesWithDuplicate), StandardCharsets.UTF_8);
+        Assert.assertEquals(expected, actual);
+    }
+
+    @Test
+    public void testSaveExternalFilesListToPreferences() throws DuplicatedJournalFileException {
+        saveFilesToPreferences();
+
+        List<String> expected = Arrays.asList(testFile1Entries, testFile3Entries, testFile4Entries,
+                testFile5EntriesWithDuplicate);
+        List<String> actual = Globals.prefs.getStringList(JabRefPreferences.EXTERNAL_JOURNAL_LISTS);
+        Assert.assertEquals(expected, actual);
+    }
+
+    @Test
+    public void testBindings() throws FileNotFoundException {
+        AbbreviationsFileViewModel testFile = new AbbreviationsFileViewModel(testFile5EntriesWithDuplicate);
+        testFile.readAbbreviations();
+        SimpleListProperty<AbbreviationsFileViewModel> testProperty = new SimpleListProperty<>(
+                FXCollections.observableArrayList(testFile));
+
+        viewModel.bindFileItems(testProperty);
+
+        Assert.assertEquals(testFile, viewModel.currentFileProperty().get());
+    }
+
+    private String createTemporaryTestFile(String name, String content) {
+        File testFile = null;
+        try {
+            testFile = tempFolder.newFile(name);
+            try (OutputStream outputStream = new FileOutputStream(testFile)) {
+                outputStream.write(content.getBytes(StandardCharsets.UTF_8));
+            }
+        } catch (IOException e) {
+            System.err.println(e.getLocalizedMessage());
+        }
+        return testFile.getAbsolutePath();
+    }
+
+    private void addAbbrevaition(Abbreviation testAbbreviation) throws DuplicatedJournalAbbreviationException {
+        viewModel.abbreviationsNameProperty().set(testAbbreviation.getName());
+        viewModel.abbreviationsAbbreviationProperty().set(testAbbreviation.getAbbreviation());
+        viewModel.addAbbreviation();
+    }
+
+    private void editAbbreviation(Abbreviation testAbbreviation) throws JabRefException {
+        viewModel.abbreviationsNameProperty().set(testAbbreviation.getName());
+        viewModel.abbreviationsAbbreviationProperty().set(testAbbreviation.getAbbreviation());
+        viewModel.editAbbreviation();
+    }
+
+    private void saveFilesToPreferences() throws DuplicatedJournalFileException {
```

Comment:
But the important thing here is that the files are getting saved to the preferences..maybe I change it to `addFourTestFileToViewModelAndPreferences`.